libraries/aws/provisioning/src/aws_iot_provisioning_serializer.c

```diff
@@ -167,7 +167,8 @@ static AwsIotProvisioningError_t _serializeCreateKeysAndCertificateRequestPayloa
                                                                    0 );
 
     /* Close the map. */
-    if( checkSerializerStatus( _pAwsIotProvisioningEncoder->closeContainer( pOutermostEncoder, &emptyPayloadEncoder ) ) == false )
+    if( checkSerializerStatus( _pAwsIotProvisioningEncoder->
+                                  closeContainer( pOutermostEncoder, &emptyPayloadEncoder ) ) == false )
```

Comment:
Is this indentation a result of uncrustify?

Reply:
Nope, I manually put a break line to limit the code length to 80 characaters. Thinking about it again, I think the limit is 120 characters